coderd/templates_test.go

```diff
@@ -665,6 +665,6 @@ func TestTemplateDAUs(t *testing.T) {
 	workspaces, err = client.Workspaces(ctx, codersdk.WorkspaceFilter{})
 	require.NoError(t, err)
 	assert.WithinDuration(t,
-		time.Now(), workspaces[0].LastUsedAt, time.Minute,
+		time.Now().UTC(), workspaces[0].LastUsedAt, time.Minute,
```

Comment:
Could this just be `database.Now()` too?

Reply:
Fixed!